pytest_selenium/drivers/cloud.py

```diff
@@ -38,5 +38,13 @@ def get_credential(self, key, env):
                 KeyError):
             value = os.getenv(env)
```

Comment:
Instead of adding `get_jenkins_credential` we could accept a list for `key`, so we can pass the potential values to look for. For example:

```python
self.get_credential('key', ['TESTINGBOT_USR', 'TESTINGBOT_KEY'])
```

Then we can just loop over these keys and return the first time we find a value. This also means we can set a precedence so `TESTINGBOT_KEY` would override `TESTINGBOT_USR`, which I think makes sense.